plugins/plugins-list-actual.json

```diff
@@ -27,5 +27,7 @@
     "zhipu",
     "glavred",
     "termef",
-    "pixabay"
+    "pixabay",
+    "idphoto",
```

Comment:
I think a good idea to sort this list by alphabet, also we can add a CI task to check that this list is alpha-sorted
It will simplify adding new plugins as list